impexp-config/src/main/java/org/citydb/config/project/kmlExporter/KmlExporter.java

```diff
@@ -455,7 +462,23 @@ public void setEmbedTexturesInGltfFiles(boolean embedTexturesInGltfFiles) {
 	}
 
 	public boolean isEmbedTexturesInGltfFiles() {
-		return embedTexturesInGltfFiles;
+		return this.embedTexturesInGltfFiles;
+	}
+
+	public void setExportGltfBinary(boolean exportGltfBinary) {
+		this.exportGltfBinary = exportGltfBinary;
+	}
+
+	public boolean isExportGltfBinary() {
+		return this.exportGltfBinary;
+	}
+
+	public void setEnableGltfDracoCompression(boolean enableGltfDracoCompression) {
+		this.enableGltfDracoCompression = enableGltfDracoCompression;
+	}
+
+	public boolean isEnableGltfDracoCompression() {
+		return this.enableGltfDracoCompression;
```

Comment:
Moved to branch [glb-draco-support](https://github.com/3dcitydb/importer-exporter/tree/glb-draco-support).